ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/monitor-sessions.md

```diff
@@ -80,6 +81,12 @@ interface Management1
 | Access Group Name | ipv4ACL |
 | Sample | 50 |
 
+### Monitor Sessions Default
+
+| Settings | Values |
+| -------- | ------ |
+| Encapsulation GRE Payload | monitor_sessions_default_encapsulation_gre.payload |
```

Comment:
Execute molecule again to reflect value of monitor_sessions_default_encapsulation_gre.payload.

Reply:
Fixed!